plugins/wiggle/src/LinearWiggleDisplay/models/model.tsx

```diff
@@ -677,23 +676,11 @@ function stateModelFactory(
          */
         async reload() {
           self.setError()
-          const aborter = new AbortController()
-          self.setLoading(aborter)
-          try {
-            const stats = await getStats(self, {
-              signal: aborter.signal,
-              ...self.renderProps(),
-            })
-            if (isAlive(self)) {
-              self.updateStats(stats)
-              superReload()
-            }
-          } catch (e) {
-            self.setError(e)
-          }
+          superReload()
```

Comment:
this is a rider change but i manually tested it and our integration tests also test it, don't need to manually do stats as autorun will do so after clearing the error